pywinauto/unittests/test_HwndWrapper.py

```diff
@@ -694,6 +695,18 @@ def testDragMouseInput(self):
         self.ctrl.drag_mouse_input("left", birds.rectangle().mid_point(), dogs.rectangle().mid_point())
         dogs = self.ctrl.GetItem(r'\Dogs')
         self.assertEquals([child.Text() for child in dogs.children()], [u'Birds', u'Dalmatian', u'German Shepherd', u'Great Dane'])
+		
+    def testVerifyEnabled(self):
+        "test for verify_enabled"
+        self.dlg.TabControl.Select(4)
+        editBox = self.dlg.EditBox.WrapperObject()
+        self.assertRaises(ElementNotEnabled, editBox.verify_enabled)
+
+    def testVerifyVisible(self):
+        "test for verify_visible"
+        editBox = self.dlg.EditBox.WrapperObject()
```

Comment:
There is no need in the implicit ```WrapperObject()``` call. Just use ```self.dlg.EditBox``` and it will be auto-resolved to the appropriate wrapper.
Why the tests are placed under ```DragAndDropTests``` ? The ```HwndWrapperTests``` looks as the more appropriate place for me.
A couple of notes about the code style. 
1.Name the new test methods according to PEP-8 recommendations even though the existing methods names are not converted yet. 
2. Use three-quote doc strings everywhere: ```"""Test for verify_visible"""```